app/Http/Controllers/Backend/Auth/AuthorizationController.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace App\Http\Controllers\Backend\Auth;
+
+use App\Models\OAuthClient;
+use App\Models\Webhook;
+use App\Repositories\OAuthClientRepository;
+use App\Rules\AuthorizedWebhookURL;
+use App\Rules\StringifiedWebhookEvents;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+use Laravel\Passport\ClientRepository;
+use Laravel\Passport\Http\Controllers\AuthorizationController as PassportAuthorizationController;
+use Psr\Http\Message\ServerRequestInterface;
+use Laravel\Passport\TokenRepository;
+use Illuminate\Support\Str;
+use League\OAuth2\Server\Exception\OAuthServerException;
+use Spatie\ValidationRules\Rules\Delimited;
+
+class AuthorizationController extends PassportAuthorizationController {
+    // most of this is based on passports original code
+    // see: https://github.com/laravel/passport/blob/11.x/src/Http/Controllers/AuthorizationController.php
+    /**
+     * Authorize a client to access the user's account.
+     *
+     * @param  \Psr\Http\Message\ServerRequestInterface  $psrRequest
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Laravel\Passport\ClientRepository  $clients
+     * @param  \Laravel\Passport\TokenRepository  $tokens
+     * @return \Illuminate\Http\Response
+     */
+    public function authorize(
+        ServerRequestInterface $psrRequest,
+        Request $request,
+        ClientRepository $_,
+        TokenRepository $tokens
```

Comment:
Note to self: can we reduce this code with parent::authorize, to maybe reduce future complications, once passport get's a major update?